Makefile

```diff
@@ -0,0 +1,42 @@
+DEL = rm -f
+MAKE = make
+SETCAP = sudo setcap CAP_NET_ADMIN+eip
+
+# Net device related
+IFUTIL = sudo ip
+DEV = udptun
+IP = 10.1.1.0
+CIDR = 24
+
+BUILD_DIR = $(CURDIR)/build
+TARGET = $(BUILD_DIR)/udp-tunnel
+
+export
+
+default: 
+	mkdir build &>/dev/null || true
```

Comment:
My fault, we can't, because `mkdir` will return non-zero value when directory exists.
At least we should use `mkdir build || true`.